INSTALL.rst

```diff
@@ -23,21 +22,29 @@ CentOS
 -------
 .. code-block:: console
 
-    $ sudo yum install python-pip python-virtualenv python-virtualenvwrapper \
+    $ sudo yum install python-virtualenv python-virtualenvwrapper \
         npm postgresql postgresql-devel libxml2-devel ImageMagick redis git \
         libxslt-devel
     $ sudo npm -g install npm
 
 For elasticsearch you can find the installation instructions on the
 `elasticsearch install page`_, and, to run the development environment, you
-will need also add the following workarounds:
+will need also to add the following workarounds:
 
 .. code-block:: console
 
     $ sudo usermod -a -G $USER elasticsearch
     $ newgrp elasticsearch  # or log out and in again
     $ sudo ln -s /etc/elasticsearch /usr/share/elasticsearch/config
 
+Arch Linux
+----------
```

Comment:
As much as I like Arch Linux, we don't have the resources to (officially) support more than one distro.